includes/footer.php

```diff
@@ -25,23 +25,19 @@
           <img height="30px" src="/assets/img/logo/sanger-tol-logo.svg" class="hide-dark">
           <img height="30px" src="/assets/img/logo/sanger-tol-logo-darkbg.svg" class="hide-light">
         </a>
-        <small class="d-block mb-3">Making awesome workflows since 2018</small>
+        <small class="d-block mb-3">Decoding earth's diversity since 2019</small>
```

Comment:
This website is about pipelines, can you include workflows word as well?

Reply:
We are going to add tools as well. Any suggestions on something the brings Tree of Life and pipelines/tools together?

Reply:
I'm personally happy with "Decoding earth's diversity since 2019". Maybe capitalise Earth ?